pkg/signature/ed25519.go

```diff
@@ -0,0 +1,37 @@
+package verification
```

Comment:
it's located in pkg/singature by there is a package 'verificaiton'. 

How about naming the package 'crypto'? Later here we can add methods like 'SignContentEd25519' so not only verification

Reply:
updated to crypto